api/wasm/cpp/proxy_wasm_impl.h

```diff
@@ -288,7 +288,7 @@ class ContextBase {
 class RootContext : public ContextBase {
 public:
   RootContext(uint32_t id, StringView root_id) : ContextBase(id), root_id_(root_id) {}
-  ~RootContext() {}
+  virtual ~RootContext() {}
```

Comment:
ContextBase's dtor is virtual so I thought I'd throw this in there

Reply:
Sure for documentation.  It is virtual anyway as per the spec.